packages/disputer/README.md

```diff
@@ -0,0 +1,26 @@
+# @uma/disputer
+
+This package contains the UMA Dispute bot reference implementation. This executable will watch an ExpiringMultiParty
+contract for liquidations and dispute any that it deems to be invalid.
+
+## Installing the package
+
+```bash
+yarn global add @uma/disputer
+```
+
+## Running the disputer
+
+The simplest way to run the disputer (with default parameters and price feeds) is:
+
+```bash
+EMP_ADDRESS=0x1234 CUSTOM_NODE_URL=your.node.url MNEMONIC="your mnemonic here" disputer --network mainnet_mnemonic
+```
+
+## Other networks and private keys
+
+Check out [the docs](https://docs.umaproject.org/developers/setup#keys-and-networks) for more options on how to plug in your private keys in different ways or use different networks.
+
+## More customization options
```

Comment:
perhaps also link to: https://docs.umaproject.org/developers/bots to give more info on setting up bots? that's pretty much the canonical docs for all things bots related.

Reply:
Good point -- will add a link to that at the top of both the liquidator and disputer!

Reply:
Done.